pom.xml

```diff
@@ -15,34 +15,31 @@
     <url>https://wiki.jenkins-ci.org/display/JENKINS/Extra+Tool+Installers+Plugin</url>

```

Comment:
BTW I like to rewrite these to the current canonical URL.

Reply:
Will do it before the release. @daniel-beck Is it already fine to reference plugins.jenkins.io instead of Wiki?

Reply:
I was rather referring to: https://wiki.jenkins.io/display/JENKINS/Extra+Tool+Installers+Plugin